packages/flutter/lib/src/material/dropdown_menu.dart

```diff
@@ -581,6 +583,9 @@ class _DropdownMenuState<T> extends State<DropdownMenu<T>> {
       }
       setState(() {
         leadingPadding = getWidth(_leadingKey);
+        if (leadingPadding != null) {
+          leadingPadding = leadingPadding! + _kLeadingIconToInputPadding;
+        }
```

Comment:
Rather than setting leadingPadding twice can you set it to a local variable first?